scripts/pyro_camera_server.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+"""
+Script to run a Pyro camera server. This should be run on the camera control computer for a
+distributed camera. The configuration for the camera is read from the pyro_camera.yaml/
+pyro_camera_local.yaml config file. The camera servers should be started after the name server,
+but before POCS.
```

Comment:
Could we have this script check for a running name server and refuse to start if so? (Can be separate PR)